pages/desktop/user.py

```diff
@@ -119,8 +119,15 @@ def click_update_account(self):
         self.selenium.find_element(*self._update_account_locator).click()
         WebDriverWait(self.selenium, self.timeout).until(lambda s: self.update_message == "Profile Updated")
 
-    def change_hide_email_state(self):
-        self.selenium.find_element(*self._hide_email_checkbox).click()
+    def hide_email_address(self):
+        email_checkbox = self.selenium.find_element(*self._email_checkbox)
+        if not email_checkbox.is_selected():
+                self.selenium.find_element(*self._email_checkbox).click()
```

Comment:
So I added a wait for the checkbox upon loading edit profile, another wait on hide_email_address to make sure it's visible- do we really need a third wait to ensure that the checkbox was checked? That will be two wait statements in one method- three if I also add one for line 124.